pkg/executables/clusterctl.go

```diff
@@ -148,12 +152,39 @@ func writeInfrastructureBundle(clusterSpec *cluster.Spec, rootFolder string, bun
 	return nil
 }
 
+func ClusterctlMoveRetryPolicy(totalRetries int, err error) (retry bool, wait time.Duration) {
+	// Exponential backoff on network errors.  Retrier built-in backoff is linear, so implementing here.
+
+	// Retrier first calls the policy before retry #1.  We want it zero-based for exponentiation.
+	if totalRetries < 1 {
+		totalRetries = 1
+	}
+	const networkFaultBaseRetryTime = 10 * time.Second
+	const backoffFactor = 1.5
+	waitTime := time.Duration(float64(networkFaultBaseRetryTime) * math.Pow(backoffFactor, float64(totalRetries-1)))
+
+	if match := connectionRefusedRegex.MatchString(err.Error()); match {
+		return true, waitTime
+	}
+	if match := ioTimeoutRegex.MatchString(err.Error()); match {
+		return true, waitTime
```

Comment:
I think illustrating some example scenarios where these errors might occur would help justify this change, i.e. "Cluster is unreachable due to X" or "Process y is not running because Z". In general, there is value to fast failure of operations and if we're retrying them it helps to explain the value added, either in comments or in the PR description.

Reply:
Added code comments describing the scenarios being protected against, primarily network disruptions between whatever's hosting clusterctl, and the source and target clusters it's moving between.  This also protects against situations where apiserver on these is temporarily not running, or becomes unresponsive during a move.